sdk/samples/iot/paho_iot_hub_pnp_component_sample.c

```diff
@@ -488,21 +492,21 @@ static void send_serial_number(void)
       publish_message.topic, publish_message.out_payload, IOT_SAMPLE_MQTT_PUBLISH_QOS);
   IOT_SAMPLE_LOG_SUCCESS(
       "Client sent `%.*s` reported property message.",
-      az_span_size(twin_reported_serial_number_property_name),
-      az_span_ptr(twin_reported_serial_number_property_name));
+      az_span_size(property_reported_serial_number_property_name),
```

Comment:
Nit - the `property_...property_name` is stuttering - do we need it twice here?  `property_reported_serial_number_name` instead.  ??